lib/src/components/NxPolicyThreatSlider/NxPolicyThreatSlider.scss

```diff
@@ -7,19 +7,24 @@
 @import '../../scss-shared/nx-variables';
 
 .nx-policy-threat-slider {
-  display: inline-block;
+  $rail-height: 8px;
+  $thumb-size: 32px;
 
-  font-size: 13px;
+  // distance from top of thumb to top of rail
+  $thumb-vertical-overflow: ($thumb-size - $rail-height) / 2;
+
+  // flex so line-height doesn't get involved in the element overall height
+  display: inline-flex;
+  margin: $nx-spacing-xs 0;
 
   // the .MuiSlider-thumb elements actually stick out of the sides of the .MuiSlider-root at their min and max
   // positions.  This padding ensures that they don't stick out of the overall box for this component
-  padding: 0 9px;
+  padding: 0 ($thumb-size / 2);
 
   .MuiSlider-root {
-    display: inline-block;
-    height: 6px;
-    padding: 9px 0;
-    width: 180px;
+    height: $rail-height;
+    padding: $thumb-vertical-overflow 0;
+    width: 200px;
```

Comment:
I'm curious about where this value comes from

Reply:
> I'm curious about where this value comes from

See the notes in the PR description.  If this isn't divisible by 10 it renders poorly on my monitor, with visible gaps between some segments.

Reply:
> > I'm curious about where this value comes from
> 
> See the notes in the PR description. If this isn't divisible by 10 it renders poorly on my monitor, with visible gaps between some segments.

I understand but the values you reference above are `I increased it by 4px from the spec'd 228 to 232` and I don't see any width values at all specified in Zeplin

Reply:
> I understand but the values you reference above are I increased it by 4px from the spec'd 228 to 232 and I don't see any width values at all specified in Zeplin

Well not written out, no.  I mean from measuring things in zeplin.

Reply:
> Well not written out, no. I mean from measuring things in zeplin.

And I just realized I misinterpreted what zeplin was saying.  I think the actual spec'd width is something like 288px or 291px.  But as I mention elsewhere, it's actually missing two segments worth of track, and adding those in would push the width up to 352px (11 * 32) which is too wide for the place that we know this needs to be usable.  Any thoughts on how we should address that?

Reply:
> But as I mention elsewhere, it's actually missing two segments worth of track, and adding those in would push the width up to 352px (11 * 32) which is too wide for the place that we know this needs to be usable

In fact, even the spec'd width is too much.  So this design really is problematic.

![image](https://user-images.githubusercontent.com/3630091/102539213-67b4f680-407b-11eb-9950-bb02f708f522.png)

Reply:
> > Well not written out, no. I mean from measuring things in zeplin.
> 
> And I just realized I misinterpreted what zeplin was saying. I think the actual spec'd width is something like 288px or 291px. But as I mention elsewhere, it's actually missing two segments worth of track, and adding those in would push the width up to 352px (11 * 32) which is too wide for the place that we know this needs to be usable. Any thoughts on how we should address that?

My thinking was that because we know that the bar length must be divisible by 10 that we either come up with 2 or 3 default sizes and hard code them, or that we allow the user to specify a value, that must be divisible by 10.

Reply:
> My thinking was that because we know that the bar length must be divisible by 10 that we either come up with 2 or 3 default sizes and hard code them, or that we allow the user to specify a value, that must be divisible by 10.

After out-of-band discussion with @glenhunter we agreed to stick to a single size but to increase it a bit while still taking care to make it fit in the existing usage. 54c99c30

Reply:
the width of the component is '290 px' but it sounds like that width won't work in this case with the included segments. I'm open to whatever solution was reached here